SPECS/bind/bind.spec

```diff
@@ -9,7 +9,7 @@
 
 Summary:        Domain Name System software
 Name:           bind
-Version:        9.16.44
+Version:        9.16.46
```

Comment:
@0xba1a, the 9.16.46 version still contains the vulnerability according to [NIST](https://nvd.nist.gov/vuln/detail/CVE-2023-50387). I'm switching to 9.16.48.